packages/sites/ortho-site/webapp/wdkCustomization/js/client/records/GroupStats.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { RecordTable } from './SequenceRecordClasses.SequenceRecordClass';
+import { RecordTableProps, WrappedComponentProps } from './Types';
+
+import eval_hist_img from '../../../images/eval-hist.png';
+
+export function RecordTable_GroupStats(
+  props: WrappedComponentProps<RecordTableProps>
+) {
+  const regularRecordTable = RecordTable(props);
+
+  const imgRef = useRef<HTMLImageElement | null>(null);
+  const [imgWidth, setImgWidth] = useState(0);
+
+  useEffect(() => {
+    setImgWidth(imgRef.current?.offsetWidth ?? 0);
+  }, [imgRef]);
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'row',
+        gap: 10,
+      }}
+    >
+      {regularRecordTable}
+      <figure
+        style={{
+          width: imgWidth || undefined,
+        }}
+      >
+        <img
+          alt="Histogram of median inter-group e-values for both core-only and core+peripheral proteins. The distributions of both peak at around 1e-20 to 1e-60 with a substantial tail out to e-values of 1e-300."
+          src={eval_hist_img}
+          ref={imgRef}
+          onLoad={() => setImgWidth(imgRef.current?.offsetWidth ?? 0)}
+        />
+        {imgWidth ? ( // only render the caption after the image is fully loaded so it doesn't shape-shift
+          <figcaption
+            style={{
+              fontSize: '90%',
+            }}
+          >
+            This histogram is provided to aid the interpretation of E-values in
+            the adjoining table. E-values have been transformed using a negative
+            logarithm, so higher significance is represented further to the
+            right.
+          </figcaption>
+        ) : null}
```

Comment:
I would set the `height` and `width` attributes of the `<img>` tag. This will cause the browser to reserve the space needed for it, while the image is loading. I think this will also allow you to remove the `width` style from `<figure>`.

Reply:
So we shouldn't bother with the dynamic width determination?

Reply:
Happy to remove it! Seems OK to hardcode the image dimensions if the image itself is hardcoded into the page!

Reply:
Removed it!

Reply:
Just seeing these comments. Hardcoding the image size is safer for a couple of reasons:
1. It reserves space on the page for the image.
2. If the image gets replaced with a huge version, it won't impact the layout.